src/Interfaces/ILiFi.sol

```diff
@@ -3,7 +3,7 @@ pragma solidity ^0.8.17;
 
 /// @title LIFI Interface
```

Comment:
do we apply new titles pattern also to interfaces?

Reply:
updated